client.go

```diff
@@ -28,8 +28,9 @@ import (
 )
 
 type EntityType string
-const(
-	PORT_GROUP EntityType = "PORT_GROUP"
+
+const (
+	PORT_GROUP     EntityType = "PORT_GROUP"
 	LOGICAL_SWITCH EntityType = "LOICAL_SWITCH"
```

Comment:
needs rebase, since the above got fixed with this commit https://github.com/eBay/go-ovn/commit/16f56680e2c066dc3394037587ca9bf24e04f646

Reply:
will close this PR and resubmit with fixes (this time from a topic branch)